docs/design/overview.md

```diff
@@ -3,26 +3,26 @@
 # oc-mirror Design Overview
 
 ## Summary 
-oc-mirror assists with lifecycle management of internet-disconnected OpenShift environments by performing differential update operations and synchronization between internet-connected and internet-disconnected environments.   
+oc-mirror assists with lifecycle management of mirror registries that support OpenShift environments by performing differential update operations and synchronizations.   
   
 ## Features
 1. Differential Synchronization of OpenShift Release Images
 2. Differential Synchronization of Operator images
-3. Synchronization of OpenShift cincinnati graph data image
+3. Synchronization of OpenShift cincinnati graph data image (planned)
 4. Synchronization of OpenShift Release images by version or channel latest
-5. Synchronization of Operator Images by full catalog and full catalog at latest operator versions. Also, allows for synchronization of individual operators by version or channel head.
+5. Synchronization of Operator Images by full catalog and full catalog at latest operator versions. Also, allows for synchronization of individual operators by version to channel(s') HEAD.
```

Comment:
Not sure why there are parentheses in `channel(s')`?

Reply:
Yea I think with the `'` it looks odd. I know typically that notation is used to convey one or more.

Reply:
I dont know how to say this in a coherent way that is close to grammatically correct. I'll try again

Reply:
I think what you wrote here is fine. I just wonder what parentheses are for? You can just move them I think.